server/index.js

```diff
@@ -80,6 +80,13 @@ export default class Server {
       const p = join(__dirname, '..', 'client', ...(params.path || []))
       await serveStatic(req, res, p)
     })
+
+    this.router.get('/_webpack/:number', async (req, res, params) => {
```

Comment:
I think here `_webpack` is the path we exposed as `publicPath` in the config.

I think in the dev mode, this is implemented by webpack dev server itself. We only need to do this on production.